.github/CONTRIBUTING.md

```diff
@@ -1,66 +1,57 @@
-Thanks for reading our contribution guidelines! What do you want to do:
+# Contribution Guidelines
+![Image of yoast](https://yoast-mercury.s3.amazonaws.com/uploads/2013/02/Yoast_Logo_Large_RGB.png)
+Before filing a bug report or a feature request, be sure to read the contribution guidelines.
 
-* [File a bug / an issue](#filing-issue)
-* [Security issues](#security)
-* [Contribute to Yoast SEO](#contribute)
+## How to use GitHub
+We use GitHub exclusively for well-documented bugs, feature requests ands code contributions. 
 
-<a name="filing-issue"></a>
-#Filing issues
+To receive free support for WordPress SEO we have the following channels:
+* [Yoast Knowledge base](http://kb.yoast.com)
+* [Support forums](https://wordpress.org/support/plugin/wordpress-seo) on WordPress.org
 
-__Please Note:__ GitHub is for bug reports and code contributions only - if you have a support question or a request for a customisation don't post here, go to our [Support Forum](http://wordpress.org/support/plugin/wordpress-seo) instead.
+If you want personal support via email, you can purchase [our premium plugins](https://yoast.com/wordpress/plugins/).
```

Comment:
If you have purchased one of [our premium plugins](https://yoast.com/wordpress/plugins/) we will give you personal support via email, please see your purchase email for details.